book/src/packaging_debs.md

```diff
@@ -10,15 +10,9 @@ This happens in Docker currently, and here's some instructions for doing it for
 3. Install the required dependencies by running `./scripts/install_ubuntu_dependencies.sh`.
 4. Building packages uses make, get a list by running `make -f ./platform/debian/Makefile help`
 
-   ➜ make -f platform/debian/Makefile help
-   debs/kanidm:
-      build a .deb for the Kanidm CLI
-   debs/kanidmd:
-      build a .deb for the Kanidm daemon
-   debs/kanidm-unixd:
-      build a .deb for the Kanidm UNIX tools (PAM/NSS, unixd and related tools) and SSH tools
-   debs/all:
-      build all the debs
+   ➜ make -f platform/debian/Makefile help debs/kanidm: build a .deb for the Kanidm CLI
+   debs/kanidmd: build a .deb for the Kanidm daemon debs/kanidm-unixd: build a .deb for the Kanidm
+   UNIX tools (PAM/NSS, unixd and related tools) and SSH tools debs/all: build all the debs
```

Comment:
```suggestion
```shell
➜ make -f platform/debian/Makefile help 
debs/kanidm: build a .deb for the Kanidm CLI
debs/kanidmd: build a .deb for the Kanidm daemon 
debs/kanidm-unixd: build a .deb for the Kanidm UNIX tools (PAM/NSS, unixd and related tools) and SSH tools 
debs/all: build all the debs
```
```